tests/test_MOcovMFile_recognizes_classdef_syntax.m

```diff
@@ -0,0 +1,81 @@
+function test_suite = test_MOcovMFile_recognizes_classdef_syntax
+    initTestSuite;
+end
+
+function fullname = tempfile(filename, contents)
```

Comment:
Sure!

I was wondering about creating a 'support' folder inside 'tests, to this and other methods that can be shared. What do you thing?